Hardware.Info/Linux/HardwareInfo.cs

```diff
@@ -184,11 +185,117 @@ public List<CPU> GetCpuList()
                 }
             }
 
+            cpu.PercentProcessorTime = GetTotalCpuUsage();
+
+            for (var i = 0; i < cpu.NumberOfLogicalProcessors; i++)
+            {
+                CpuCore core = new CpuCore
+                {
+                    Name = i.ToString(), 
+                    PercentProcessorTime = GetCpuCoreUsage(i)
+                };
+                cpu.CpuCoreList.Add(core);
+            }
+
             cpuList.Add(cpu);
 
             return cpuList;
         }
 
+        private static UInt64 GetTotalCpuUsage()
```

Comment:
It would be possible to merge both methods by providing the core number and if it's empty then take the total CPU usage. Depends on the personal preferences here...

Reply:
Yes, please merge both methods